guides/deployment/heroku.md

```diff
@@ -171,17 +145,21 @@ Now, let's tell Phoenix to use our Heroku URL and enforce we only use the SSL ve
 url: [host: "example.com", port: 80],
 ```
 
-... and replace it with this (don't forget to replace `mysterious-meadow-6277` with your application name):
+... and replace it with this:
 
 ```elixir
-url: [scheme: "https", host: "mysterious-meadow-6277.herokuapp.com", port: 443],
+url: [scheme: "https", host: "", port: 443],
```

Comment:
Can the host be actually empty? I think you have to fill it in.

Reply:
Ah yes - my fault. I only checked with Routes.page_path and not page_url. Correction coming ....